src/operations/rgb/descriptor_wallet.rs

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
What is the purposed of this? Is al commented out

Reply:
It's in case I needed to embed the descriptor wallet, and it seems like I will need to.